app/src/bcsc-theme/navigators/MainStack.tsx

```diff
@@ -194,7 +195,16 @@ const MainStack: React.FC = () => {
           component={MandatoryUpdate}
           options={{
             ...getDefaultModalOptions(t('BCSC.Title')),
-            gestureEnabled: false, // Disable swipe to dismiss
+            gestureEnabled: false,
```

Comment:
thought: I added this, but I realized I didnt test if on android you can navigate away from this screen with the swipe horizontal motion. 

I know this prevents the swipe down from closing, but uncertain about "back".